package.json

```diff
@@ -9,7 +9,7 @@
     "check:fix": "./scripts/check-filenames.bash --fix",
     "lint": "for f in ./nominees/*.json ./digitalpublicgoods/*.json; do jsonlint \"$f\" > tempfile.tmp; if diff $f tempfile.tmp; then echo \"Linting of $f passed\"; else echo \"Linting of $f failed\" && exit 1; fi; done",
     "lint:fix": "for f in ./nominees/*.json ./digitalpublicgoods/*.json; do echo \"$f\"; jsonlint \"$f\" -i; echo >> \"$f\"; done",
-    "order": "node scripts/order-fields.js",
+    "order": "node scripts/order-fields.js --color=always",
```

Comment:
I should write this force color tag somewhere. This is dope @lacabra

Reply:
Sorry to disappoint, but I am afraid this is not as dope as it looks because my understanding is that this is specific to the `colors` package, read [this section](https://www.npmjs.com/package/colors#enablingdisabling-colors) (because we are passing a variable to that specific package); but it doesn't generalize to all NodeJS code.